spec/helpers/countries_helper_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe CountriesHelper, type: :helper do
+  describe "#countries" do
+    def get_countries(country_code: nil)
```

Comment:
same here, we should not make an actual call to the service but mock its response instead